custom_components/vikunja/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "domain": "vikunja",
   "name": "Vikunja",
-  "version": "0.12",
+  "version": "0.13",
```

Comment:
Given you have other plans for configuration changes, what sort of timescale are you considering for wanting to make those proposed changes?

If its short-ish then we can just wait and release the lot as one?